### PR TITLE
docs(readme): add instructions on how to get the webapp to download and run

### LINKED ISSUE #278: Add download docs
Add docs that explain how to get the qri frontend to work

### PR DESCRIPTION
**Pull request by ramfox**
closes #278 

Definitely not detailed enough yet, I don't think. This might be a good spot for a video showing/explaining what needs to be done?

### CODE REVIEW

**Review by b5 — APPROVED** [review]
🗻 🚵‍♀️ ⛰

File: readme.md [hunk 1]
```diff
@@ -15,7 +15,55 @@
...
 
 ## Usage
-[Download the latest release!](https://qri.io/#installation)
+
+### Installing Dependencies
+
+Before you can build the Qri frontend, you must install and configure the following dependencies on your machine:
+
+* [Qri](http://www.qri.io): The qri server and command line client.
+  If you didn't want a pretty webapp to interact with, you would only need to install Qri itself. Head over to the [ri github page](https://www.github.com/qri-io/qri) and follow the instructions under 'Building From Source' to download and install Qri.
```
Comment (lightandluck) [inline readme.md]: Typo: `[ri github..` -> `[Qri github`

_Sorry, just lurking, back to my cave now_ =P

Comment (b5) [reply readme.md]: nice catch @lightandluck! also, 👋!

Comment (lightandluck) [reply readme.md]: 👋😁

Comment (ramfox) [reply readme.md]: woooo thanks `:)`

**Comment by b5** [comment]
@lightandluck if you ever decide to take a crack at getting our frontend up & running feel free to reach out / file an issue with any questions, and we'd be happy to take some time to walk through it. Also, if you see parts of this codebase that seem useful or techniques you'd like explained in greater detail, just holler :)

**Comment by lightandluck** [comment]
@b5 Yup, definitely on my list of want-to-dos to play around with Qri. I'll take you up on that offer when I have the chance. Thanks!

**Pull request merged**